contrib/pg_tde/pg_tde--1.0-rc.sql

```diff
@@ -324,6 +324,16 @@ LANGUAGE C
 AS 'MODULE_PATHNAME';
 REVOKE ALL ON FUNCTION pg_tde_verify_default_key() FROM PUBLIC;
 
+CREATE FUNCTION pg_tde_delete_key()
+RETURNS VOID
+LANGUAGE C
+AS 'MODULE_PATHNAME';
+
+CREATE FUNCTION pg_tde_delete_default_key()
+RETURNS VOID
+LANGUAGE C
+AS 'MODULE_PATHNAME';
```

Comment:
Anders introduced a new pattern where we revoke execute directly after creating a C function. I think we should follow that pattern here too.